### ISSUE TITLE
bundletool failed: Failed to run on batchmode appbundle build

### ISSUE THREAD

**Issue by joonjoonjoon** [google/play-unity-plugins#106]
Seeing this in the log of a batchmode build

```

bundletool failed: Failed to run '/Applications/Unity/Hub/Editor/2020.3.4f1/PlaybackEngines/AndroidPlayer/OpenJDK/bin/java -jar "/Users/joon/git/inua/Assets/GooglePlayPlugins/com.google.android.appbundle/Editor/Tools/bundletool-all.jar" build-bundle --overwrite --config="/var/folders/8c/1lkfpptx58vfghggl5t8f48c0000gn/T/BundleConfig.json" --modules=/var/folders/8c/1lkfpptx58vfghggl5t8f48c0000gn/T/play-unity-build/jor9ixr0.2dg/scenes/scenes.zip,/var/folders/8c/1lkfpptx58vfghggl5t8f48c0000gn/T/play-unity-build/jor9ixr0.2dg/base/base.zip --output="inua.aab" --metadata-file=com.android.tools.build.libraries/dependencies.pb:/var/folders/8c/1lkfpptx58vfghggl5t8f48c0000gn/T/play-unity-build/jor9ixr0.2dg/base/source/BUNDLE-METADATA/com.android.tools.build.libraries/dependencies.pb'
stdout:

stderr:
[BT:1.5.0] Error: null
java.lang.NullPointerException
	at java.nio.file.Files.provider(Files.java:97)
	at java.nio.file.Files.notExists(Files.java:2434)
	at com.android.tools.build.bundletool.commands.BuildBundleCommand.execute(BuildBundleCommand.java:260)
	at com.android.tools.build.bundletool.BundleToolMain.main(BundleToolMain.java:71)
	at com.android.tools.build.bundletool.BundleToolMain.main(BundleToolMain.java:47)
```

I am using this command to build the app bundle

 ```
 public static void Build()
    {
        var assetPackConfig = new AssetPackConfig();
        assetPackConfig.AddAssetBundle(AssetBundleDirectory + "scenes", AssetPackDeliveryMode.FastFollow);
       
        //CreateAssetPackConfig();
        var buildPlayerOptions = AndroidBuildHelper.CreateBuildPlayerOptions(AabFilePath);

        if (!Bundletool.BuildBundle(buildPlayerOptions, assetPackConfig))
        {
            throw new Exception("Asset Delivery Demo build failed");
        }
    }
```

**Comment by joonjoonjoon** [comment]
I downgraded to release 1.3.0 from 1.4.0 and the bug no longer shows up, with otherwise identical code.

I google translated one of the google results when trying to find out what the error means and they suggested the downgrade: https://baba-s.hatenablog.com/entry/2021/04/27/150000

**Comment by thanwah** [comment]
Thanks @joonjoonjoon for filing this issue and for describing your workaround.

I believe this is caused by https://github.com/google/bundletool/issues/214, and I've reached out to the bundletool owner to find out when a fix will be released.

**Comment by joonjoonjoon** [comment]
this is likely the same issue as https://github.com/google/play-unity-plugins/issues/102

**Comment by thanwah** [comment]
Now fixed at HEAD by updating bundletool to 1.6.1.
